cmd/util/cmd/rollback-executed-height/cmd/rollback_executed_height_test.go

```diff
@@ -213,40 +179,15 @@ func TestReExecuteBlockWithDifferentResult(t *testing.T) {
 		)
 		require.NotNil(t, es)
 
-		// prepare data
 		executableBlock := unittest.ExecutableBlockFixtureWithParent(
 			nil,
-			genesis) // make sure the height is higher than genesis
+			genesis,
+			&unittest.GenesisStateCommitment)
 		header := executableBlock.Block.Header
-		executionReceipt := unittest.ExecutionReceiptFixture()
-		executionReceipt.ExecutionResult.BlockID = header.ID()
-		cdp := make([]*flow.ChunkDataPack, 0, len(executionReceipt.ExecutionResult.Chunks))
-		for _, chunk := range executionReceipt.ExecutionResult.Chunks {
-			cdp = append(cdp, unittest.ChunkDataPackFixture(chunk.ID()))
-		}
-		endState, err := executionReceipt.ExecutionResult.FinalStateCommitment()
-		require.NoError(t, err)
-		blockEvents := unittest.BlockEventsFixture(header, 3)
-		// se := unittest.ServiceEventsFixture(2)
-		se := unittest.BlockEventsFixture(header, 8)
-		tes := unittest.TransactionResultsFixture(4)
-
-		err = headers.Store(header)
-		require.NoError(t, err)
 
```

Comment:
@zhangchiqing I updated this test but it keeps failing, I think there was probably something in the logic of retry and not the test itself, I need your help debugging this.